src/views/forms/NewMarketForm/v7/NewMarketSuccessStep.tsx

```diff
@@ -15,25 +17,39 @@ import { Icon, IconName } from '@/components/Icon';
 import { getDisplayableTickerFromMarket } from '@/lib/assetUtils';
 
 type NewMarketSuccessStepProps = {
-  onActionClicked?: () => void;
+  onActionClicked?: (marketId: string) => void;
   onLaunchAnotherMarket: () => void;
+  onResubscribeMarketId?: (marketId: string) => void;
   tickerToAdd: string;
   transactionUrl: string;
 };
 
 export const NewMarketSuccessStep = ({
   onActionClicked,
   onLaunchAnotherMarket,
+  onResubscribeMarketId,
   tickerToAdd,
   transactionUrl,
 }: NewMarketSuccessStepProps) => {
   const stringGetter = useStringGetter();
   const match = useMatch(`${AppRoute.Trade}/:marketId`);
   const { marketId } = match?.params ?? {};
 
+  useEffect(() => {
+    return () => {
```

Comment:
wait what's the logic here? When the url changes (how?) we resubscribe? Doesn't that subscription happen somewhere else?

Reply:
Removed the jank useEffect cleanup. This semi-worked but was overly confusing